.github/workflows/stalebot.yml

```diff
@@ -20,7 +20,8 @@ jobs:
         stale-pr-label: "Stale"
         exempt-pr-labels: "Keep" # a "Keep" label will keep the PR from being closed as stale
         days-before-pr-stale: 730 # when the PR is considered stale
-        days-before-pr-close: 60 # when the PR is closed by the bot,
-        days-before-issue-stale: 730 # prevents issues from being tagged by the bot
-        days-before-issue-close: 60 # prevents issues from being closed by the bot
+        days-before-pr-close: 30 # when the PR is closed by the bot
```

Comment:
can we shorten this to 1 week or lesser please?

Reply:
done